apps/wallet/src/ui/app/zustand/createSelectors.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { type StoreApi, type UseBoundStore } from 'zustand';
+
+type WithSelectors<S> = S extends { getState: () => infer T }
+	? S & { use: { [K in keyof T]: () => T[K] } }
+	: never;
+
+export const createSelectors = <S extends UseBoundStore<StoreApi<object>>>(_store: S) => {
```

Comment:
tbh I wouldn't add this, just manually create the selectors.

Reply:
removed this to use selectors directly